### DISCUSSION TITLE
How to add a formula for libavif?

### DISCUSSION THREAD

**Discussion by y-a-v-a** [Homebrew/discussions#1789] [Everyday usage]
Hi I'm using brew for several years now, for several projects.
I was wondering what steps one would have to make to add a formula for libavif: https://github.com/AOMediaCodec/libavif as there doesn't seem to be one yet.
I would like to have this formula in order to extend the libgd2 formula (named `gd`: https://github.com/Homebrew/homebrew-core/blob/2badf0f3f0bdf0ad5dbaa3c33462c6caa5a65d4a/Formula/gd.rb) so I can make use of libavif in the latest version of libgd2 (2.3.2).
Also, formula gd could be extended with the use of libheif ( https://github.com/Homebrew/homebrew-core/blob/1a68145a11b216a5fbbbf2bf1cc84dba9679f3ba/Formula/libheif.rb ).

Cheers Vincent - trying to maintain https://github.com/y-a-v-a/node-gd

**Comment by nandahkrishna** [comment]
You could create a pull request to add the formula. Here are some docs to help you get started:
* [Contributing Guide](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)
* [Formula Cookbook](https://docs.brew.sh/Formula-Cookbook)
* [How to open a Homebrew PR](https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request)
